core/trino-main/src/test/java/io/trino/sql/query/TestFilterInaccessibleColumns.java

```diff
@@ -160,7 +160,7 @@ public void testRowFilterOnNotAccessibleColumn()
     }
 
     @Test
-    public void testRowFilterOnNotAccessibleColumnKO()
+    public void testRowFilterWithUserOnNotAccessibleColumn()
```

Comment:
What does "with user" mean? Did you mean "unprivileged user"?

"NotAccessible" -> "inaccessible"